debian/control

```diff
@@ -0,0 +1,12 @@
+Source: ocf-atool
+Section: python
+Priority: extra
+Maintainer: Open Computing Facility <help@ocf.berkeley.edu>
+Build-Depends: debhelper (>= 9), python3, dh-virtualenv, dh-systemd (>= 1.5)
+Standards-Version: 3.9.5
+
+Package: ocf-atool
+Architecture: any
```

Comment:
There are, consider all the Python modules that we use (most of them pulled in indirectly by ocflib) that have a native component.